jdi-light/src/main/java/com/epam/jdi/light/elements/init/UIFactory.java

```diff
@@ -42,11 +42,11 @@ public static UIElement element(By byLocator) {
 
     public static WebList list(String locator) {
         return locator.matches("[A-Z].*]")
-                ? new WebList().setName(locator)
+                ? new WebList(1).setName(locator)
```

Comment:
1 should be a default behaviour so we don't need to pass it inside (you can write new constructor like WebList(){ this(1); }